ui/src/views/dashboard/UsageDashboard.vue

```diff
@@ -143,7 +143,9 @@ export default {
         if (newValue && newValue.id && (!oldValue || newValue.id !== oldValue.id)) {
           this.fetchData()
         } else if (store.getters.userInfo.roletype !== 'Admin') {
```

Comment:
```suggestion
        } else if (store.getters.userInfo.roletype !== 'Admin' && !store.getters.logoutFlag) {
```
Just a minor suggestion to decrease indentation.